Gruntfile.js

```diff
@@ -10,24 +10,26 @@ module.exports = function(grunt) {
     concat: {
       dist: {
         options: {
-          banner: "(function(){\n",
-          footer: "}());\n"
+          banner: '(function(exports, crypto, Promise, Error, Uint8Array, undefined){\n"use strict";\n\n// supporting Safari and its vendor prefix\nif(!crypto.subtle) crypto.subtle = crypto.webkitSubtle;\n',
```

Comment:
ok, I removed the undefined redefinition and replace === undefined and !== undefined with typeof check